src/adagrams.js

```diff
@@ -1,15 +1,141 @@
+const letterPoolObj = {
+  A: 9, 
+  B: 2, 
+  C: 2, 
+  D: 4, 
+  E: 12, 
+  F: 2, 
+  G: 3, 
+  H: 2, 
+  I: 9, 
+  J: 1, 
+  K: 1, 
+  L: 4, 
+  M: 2, 
+  N: 6, 
+  O: 8, 
+  P: 2, 
+  Q: 1, 
+  R: 6, 
+  S: 4, 
+  T: 6, 
+  U: 4, 
+  V: 2, 
+  W: 2, 
+  X: 1, 
+  Y: 2, 
+  Z: 1
+}
+
 export const drawLetters = () => {
   // Implement this method for wave 1
+  let letterList = [];
+  let letterPoolCopy = letterPoolObj;
+
+  for (let i = letterList.length; i < 10; ++i) {
+    let objKeys = Object.keys(letterPoolObj);
+    let randLetter = objKeys[Math.floor(Math.random() * objKeys.length)];
+    if (letterPoolCopy[randLetter] > 0){
+        letterList.push(randLetter);
+        letterPoolCopy[randLetter] -= 1;
+    }
```

Comment:
To accurately reflect drawing from a pool that represents a distribution of letters represented by LETTER_POOL, you'd need to update this logic too to check that you have a remaining letter in you available letters array and remove the letter from the array after it gets added to letterList.